packages/debug-utils/index.js

```diff
@@ -369,7 +369,11 @@ var DebugUtils = {
       return shortcuts.map(DebugUtils.formatCommandDescription).join(", ");
     });
 
-    var suffix = [""];
+    var suffix = [
+      "",
+      "Docs: https://trufflesuite.com/docs/truffle/how-to/debug-test/use-the-truffle-debugger/",
```

Comment:
Thoughts on better presentation? , Maybe:

```javascript
var suffix = [
  "",
  "Docs:",
  "https://trufflesuite.com/docs/truffle/how-to/debug-test/use-the-truffle-debugger/",
  ""
];
```

Reply:
Yeah this is likely a good idea. Hitting approve anyway because I think it's OK to merge as-is without this change. :)

Reply:
OK, I'll leave that out for now.